src/Notifiers/Slack/SlackNotifier.php

```diff
@@ -0,0 +1,324 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Notifiers\Slack;
+
+use App\Entity\Conference;
+use App\Entity\Submit;
+use App\Entity\Talk;
+use App\Repository\ConferenceRepository;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class SlackNotifier
+{
+    private string $webHookUrl;
+    private string $env;
+    private HttpClientInterface $httpClient;
+    private RouterInterface $router;
+    private ConferenceRepository $conferenceRepository;
+    private SlackBlocksBuilder $slackBlocksBuilder;
+
+    public function __construct(string $webHookUrl, string $env, HttpClientInterface $httpClient, RouterInterface $router, ConferenceRepository $conferenceRepository, SlackBlocksBuilder $slackBlocksBuilder)
+    {
+        $this->webHookUrl = $webHookUrl;
+        $this->env = $env;
+        $this->httpClient = $httpClient;
+        $this->router = $router;
+        $this->conferenceRepository = $conferenceRepository;
+        $this->slackBlocksBuilder = $slackBlocksBuilder;
+    }
+
+    /** @param array<Conference> $newConferences */
+    public function sendDailyNotification(array $newConferences): void
+    {
+        $dailyBlocks = $this->buildDailyBlocks($newConferences);
+
+        $this->notify(['blocks' => $dailyBlocks]);
+    }
+
+    /**
+     * @param array<Conference> $newConferences
+     *
+     * @return array<array>
+     */
+    public function buildDailyBlocks(array $newConferences): array
+    {
+        $conferencesBlocks = $this->buildConferencesBlocks($newConferences);
+        $endingCfpsBlocks = $this->buildCfpsBlocks();
+
+        return [
+            ...$conferencesBlocks,
+            ...$endingCfpsBlocks,
+            $this->slackBlocksBuilder->buildDivider(),
```

Comment:
Why are you using variables for the first two, but not for the last one? It's not really consistant :p